e2e_test/source/basic/ddl.slt

```diff
@@ -1,3 +1,45 @@
+statement error
+create source s;
+----
+db error: ERROR: Failed to run the query
+
+Caused by:
+  sql parser error: Expected description of the format, found: ; at line:1, column:17
+Near "create source s"
+
+
```

Comment:
@StrikeW please help adding an example about CDC source with specified dbz settings here

Reply:
Here is an example to config the connector to start w/o init snapshot.
```
create table tt3 (v1 int primary key, v2 timestamptz)
  with (
    connector = 'postgres-cdc',
    hostname = 'localhost',
    port = '8432',
    username = 'myuser',
    password = '123456',
    database.name = 'mydb',
    schema.name = 'public',
    table.name = 'tt3',
    slot.name = 'tt3_slot',
    debezium.snapshot.mode = 'never'
  );
  ```